presto-hive/src/main/java/com/facebook/presto/hive/RcFileFileWriterFactory.java

```diff
@@ -139,8 +140,14 @@ else if (ColumnarSerDe.class.getName().equals(storageFormat.getSerDe())) {
                 });
             }
 
+            Callable<Void> outputFileDeleteRoutine = () -> {
```

Comment:
you can also put the try catch here and remove it from `RcFileFileWriter.java` where you ignore it.